### ISSUE TITLE
dashboard_EXT - TypeError

### ISSUE THREAD

**Issue by master3395** [dbm-network/mods#513]
This comes up if i try a clean install of the newly merged Dashboard.
This is what happens if i go to http://localhost:3000/dashboard/callback
![image](https://user-images.githubusercontent.com/4199669/89233964-1e439d80-d5eb-11ea-9635-ce567b83dd1a.png)

**Comment by RediPanda** [comment]
Is it okay if you provided us with your config file so we know if the features variable is declared or not?

**Comment by master3395** [comment]
Here is the config file (Removed tokenSecret and clientSecret.)

[config.txt](https://github.com/dbm-network/mods/files/5022136/config.txt)


> 
> 
> Is it okay if you provided us with your config file so we know if the features variable is declared or not?

**Comment by RediPanda** [comment]
Yeah, no. That's not really a bug. You've sort of removed a variable that one of the EJS files read off.

If you don't want the features, here's the JSON file.

```json
{
  "port": "3000",
  "isBotSharded": false,
  "tokenSecret": "SECRET",
  "clientSecret": "SECRET",
  "callbackURL": "http://localhost:3000/dashboard/callback",
  "owner": ["80357836940382208"],
  "inviteLink": "/dashboard/@me",
  "supportServer": "https://discord.gg/somewhere",
  "introText": "You can replace this text with whatever you want. After you do that this will automatically show up on your website!",
  "footerText": "You can replace this text with whatever you want. After you do that this will automatically show up on your website!",
  "theme": "default",
  "isGlitch": false,
  "navItems": [
    { "name": "Home Page", "link": "/" },
    { "name": "Dashboard", "link": "/dashboard/@me" },
    { "name": "Admin Panel", "link": "/dashboard/admin" }
  ],
  "features": []
}
```

Let me know if it's still providing any issues, but for now I'll remove the bug tag.

**Comment by master3395** [comment]
I haven't removed 1 line in the config file.
I only took the whole extension folder and moved the correct folder into extensions and the root json for the extensions folder.
And added the tokens it requires.
If you find an issue, you should look in the template it self.
Not in my config.

> 
> 
> Yeah, no. That's not really a bug. You've sort of removed a variable that one of the EJS files read off.
> 
> If you don't want the features, here's the JSON file.
> 
> ```json
> {
>   "port": "3000",
>   "isBotSharded": false,
>   "tokenSecret": "SECRET",
>   "clientSecret": "SECRET",
>   "callbackURL": "http://localhost:3000/dashboard/callback",
>   "owner": ["80357836940382208"],
>   "inviteLink": "/dashboard/@me",
>   "supportServer": "https://discord.gg/somewhere",
>   "introText": "You can replace this text with whatever you want. After you do that this will automatically show up on your website!",
>   "footerText": "You can replace this text with whatever you want. After you do that this will automatically show up on your website!",
>   "theme": "default",
>   "isGlitch": false,
>   "navItems": [
>     { "name": "Home Page", "link": "/" },
>     { "name": "Dashboard", "link": "/dashboard/@me" },
>     { "name": "Admin Panel", "link": "/dashboard/admin" }
>   ],
>   "features": []
> }
> ```
> 
> Let me know if it's still providing any issues, but for now I'll remove the bug tag.

**Comment by shootdot** [comment]
This happens to me also....
Even I used the new Dashboard which was updated yesterday and I get this same error.
![image](https://user-images.githubusercontent.com/55920587/89411786-0da03e00-d737-11ea-8849-8f1466bb8e38.png)

**Comment by master3395** [comment]
I noticed if you update the config file manually in project\extensions\dashboard_EXT\config.js
It starts, but if you go into DBM, and update it from there, it makes the error shown in my first image at the top.
Below is an image that shows the difference in the config file.

1 shows how broken it gets if you update it in DBM, and 2 shows the working file by RediPanda.
![image](https://user-images.githubusercontent.com/4199669/89580924-6c001600-d836-11ea-9132-0312ebc05b04.png)

I also noticed that if you go into "Admin Panel"
And try to update "Web Panel", this happens.

![1](https://user-images.githubusercontent.com/4199669/89581404-34459e00-d837-11ea-9d65-7eff772bd5a8.JPG)
Clicking "Save settings"
Takes you to "http://localhost:3000/api/admin/web"
![2](https://user-images.githubusercontent.com/4199669/89581411-390a5200-d837-11ea-9ccd-8186bdf473a4.JPG)
